Webpack doesn't automatically ignore all Node deps when bundling.

After using [0.1.8](https://github.com/electric-sql/pglite/releases/tag/0.1.8) successfully, I'm now getting the following error while trying out these alpha releases:

- [v0.2.0-alpha.3](https://github.com/electric-sql/pglite/releases/tag/v0.2.0-alpha.3)
- [v0.2.0-alpha.4](https://github.com/electric-sql/pglite/releases/tag/v0.2.0-alpha.4)

```
./node_modules/@electric-sql/pglite/dist/index.js:1:21817
Module not found: Can't resolve 'stream/promises'
```

I'm doing this on a Next.js project v `13.5.3`. My webpack config on `next.config`:

```
webpack: (config, options) => {
	config.module.rules.push({
		test: /\.+(js|jsx|mjs|ts|tsx)$/,
		use: options.defaultLoaders.babel,
		// @ts-ignore
		include: fileURLToPath(import.meta.resolve('@electric-sql/pglite')),
		type: 'javascript/auto'
	});

	config.module.rules.push({
		test: /\.sql$/,
		// This is the asset module.
		type: 'asset/source'
	});

	if (!options.isServer) {
		config.resolve.fallback = { fs: false, module: false, path: false };
	}

	return config;
}
```

I'm not sure if I'm doing something wrong here. Please help!

--- a/packages/pglite/package.json
+++ b/packages/pglite/package.json
@@ -88,5 +88,17 @@
     "tsup": "^8.1.0",
     "tsx": "^4.7.1",
     "typescript": "^5.3.3"
+  },
+  "browser": {
+    "fs": false,
+    "path": false,
+    "url": false,
+    "zlib": false,
+    "stream": false,
+    "stream/promises": false,
+    "crypto": false,
+    "ws": false,
+    "child_process": false,
+    "module": false
   }
 }

